docs/content_management/collaborative_editing/session_api.md

```diff
@@ -0,0 +1,93 @@
+---
+description: You can use the PHP API to create new session, update existing one, find or delete it, and add or remove participants.
+---
+
+# Session API
+
+[`SessionService`](../../api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-SessionService.html) enables you work with the collaborative session, for example, create a new one, update or delete existing one, and add or remove new participants to join collaborative session.
+
+## Create session
+
+You can create new collaboration session with given id with [`SessionService::createSession`](../../api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-SessionService.html#method_createSession):
+
+``` php
+    $this->innerService->createSession($createStruct);
+```
+## Get session
+
+You can return existing collaboration session with [`SessionService::getSession`](../../api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-SessionService.html#method_getSession):
```

Comment:
```suggestion
You can retrieve an existing collaboration session with [`SessionService::getSession`](../../api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-SessionService.html#method_getSession):
```
Or `get` as in the header

I wouldn't use return as it's a keyword with special meaning in programming (you "return a value from within a function", for example)